server/handlers/walletHandler/schemas.js

```diff
@@ -4,6 +4,8 @@ const TrustRelationshipEnums = require('../../utils/trust-enums');
 const walletGetQuerySchema = Joi.object({
   limit: Joi.number().required(),
   offset: Joi.number().integer(),
+  sortField: Joi.string(),
+  sortOrder: Joi.string(),
```

Comment:
let's make this restricted to asc and desc

Reply:
@OlhaD  It may be better to rename these to `sort_by` and `order`, to match with the current version of the API, which already has these options implemented.
Also, this PR should be merged into the production branch `v1.10`, right?

The current version doesn't support sorting by token amount either, so we should create an issue for that as well.